### PR TITLE
Run strategies: RerunOnFailure is now immutable

### PR DESCRIPTION
**Pull request by jean-edouard**

### CODE REVIEW

**Comment by jean-edouard** [comment]
Finally managed to make tables page-wide, that had been bothering me for a long time!

File: docs/virtual_machines/run_strategies.md [hunk 1]
```diff
@@ -45,22 +45,22 @@ The `start`, `stop` and `restart` methods of virtctl will invoke their
 respective subresources of VirtualMachines. This can have an effect on
 the runStrategy of the VirtualMachine as below:
 
-<table>
+<table style="width: 100%">
```
Comment (fossedihelm) [inline docs/virtual_machines/run_strategies.md:48]: just FYI:
```
<table style="width: 100%, display: inline-table">
```
should work without adding any style to the `thead` and `tbody` elements

**Review by fossedihelm — APPROVED** [review]
Thanks @jean-edouard
/hold
to prevent merge before https://github.com/kubevirt/kubevirt/pull/10639 is merged

File: docs/virtual_machines/run_strategies.md (hunk 1 above)
Comment (jean-edouard) [reply docs/virtual_machines/run_strategies.md]: Indeed! :)

**Pull request merged**